src/packages/@ncigdc/utils/analytics/performanceTracker.js

```diff
@@ -0,0 +1,34 @@
+const startTimes = {};
+export const timer = {
+  time: label => (startTimes[label] = performance.now()),
```

Comment:
If you go with my suggestion for line 9, It might also be good to at least warn if startTimes[label] already exists.